test/refresh.spec.js

```diff
@@ -5,7 +5,9 @@ require('mocha');
 const chai = require('chai');
 const sinon = require('sinon');
 const expect = chai.expect;
-const AuthTokenRefresh = require('../lib/refresh.js');
```

Comment:
Distinguishing the constructor name from the instance name (following the same convention as Passport).